npm/webpack-dev-server/src/patches.d.ts

```diff
@@ -1,6 +0,0 @@
-/**
```

Comment:
This is no longer necessary (and causes a type conflict if present) now that electron 18 is resolving to @types/node 16